scripts/issue/642/adjust_bpt_bootstraps.py

```diff
@@ -0,0 +1,34 @@
+from great_ape_safe import GreatApeSafe
+from helpers.addresses import r
+
+
+def main():
+    safe = GreatApeSafe(r.badger_wallets.treasury_ops_multisig)
+    voter = GreatApeSafe(r.badger_wallets.treasury_voter_multisig)
+
+    bpt_aura = safe.contract(r.balancer.bpt_33_grav_33_weth_33_aura)
+    bpt_aurabal = safe.contract(r.balancer.bpt_33_grav_33_weth_33_aurabal)
+
+    safe.init_balancer()
+    safe.init_aura()
+    tokens = [
+        bpt_aura,
+        bpt_aurabal,
+        r.treasury_tokens.AURA,
+        r.treasury_tokens.AURABAL,
+        r.treasury_tokens.WETH,
+        r.sett_vaults.graviAURA,
+        safe.aura.booster.poolInfo(safe.aura.get_pool_info(bpt_aurabal)[0])[3]
+    ]
+    safe.take_snapshot(tokens)
+    voter.take_snapshot(tokens)
+
+    safe.balancer.unstake_all_and_withdraw_all(
+        pool=bpt_aura,
+        pool_type='non_stable',
+        destination=r.badger_wallets.treasury_voter_multisig
+    )
+    safe.aura.deposit_all_and_stake(bpt_aurabal)
```

Comment:
on my end, `assert self.booster.depositAll(pool_id, stake).return_value == True` check fails but deposit goes through successfully without the check

Reply:
what is `type` or value of the `.return_value` on your side?

Reply:
returns `None`